src/igor/sync.go

```diff
@@ -82,14 +78,38 @@ func runSync(cmd *Command, args []string) {
 }
 
 func syncArista() {
+	// first get ground truth
+	gt, err := networkVlan()
+	if err != nil {
+		log.Fatal("Unable to acquire VLAN ground truth from arista")
+	}
 	// TODO: probably shouldn't iteration over .M directly
 	for _, r := range igor.Reservations.M {
 		if !r.IsActive(igor.Now) {
 			continue
 		}
 
+		w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
```

Comment:
We should not be creating a new writer in every iteration of this loop. Please move out of the loop.

Reply:
tabwriter moved to outside the loop